src/components/Home/WhatsNewModal/styles.module.css

```diff
@@ -0,0 +1,155 @@
+.container {
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  z-index: 11;
+
+  &:hover {
+    cursor: pointer;
+  }
+
+  &Open {
+    height: 100%;
+    width: 100%;
+    background: rgba(132, 142, 148, 0.5);
+  }
+
+  @media (--xs-scr) {
+    display: none;
+  }
+}
+
+.button {
+  @mixin button-big;
+
+  position: absolute;
+  box-sizing: border-box;
+  overflow: hidden;
+  white-space: nowrap;
+  left: 30px;
+  bottom: 30px;
+  z-index: 3;
+  width: 203px;
+  height: 62px;
+  padding: 16px 24px;
+  background: linear-gradient(
+    102.58deg,
+    #13adc7 22.89%,
+    #17d5f5 24.16%,
+    #13adc7 25.62%
+  );
+  background-size: 600% auto;
+  color: #fff;
+  border: none;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: 99px;
+  font-family: var(--font-brandon);
+  transition: all 0.25s ease-in-out;
+  animation: gradient 3s infinite ease-in-out;
```

Comment:
I feel like the animation for the blue button could be better, but I'm not sure how to improve it. 🤔

Reply:
I think animation on the button would be cool and draw attention to it.  Also when I looked at the Heroku on desktop.  It feels like the button is a bit too far down on the page.  Might be better up a skosh, into the first section, or even on the line between the first and second section.  That might call attention to it more.
![image](https://user-images.githubusercontent.com/34289199/142506532-5c3d217c-e831-4fef-9a37-c385c80adee1.png)

Reply:
We could move the button higher up but we'd have to move the button lower back to the corner of the screen when the user starts scrolling. That may end up looking glitchy 🤔

Reply:
Ok.  No worries then.  Today when Dave shared in our meeting, it looked good.  👍 May be my settings.